releasenotes/notes/keep-initial-resets-a5f75cb16c8edb57.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
```

I think this is an upgrade note, because it is more like a warning that, after updating, your results might be different.
```suggestion
upgrade:
```